assembl/static2/js/app/components/administration/resourcesCenter/manageResourcesForm.jsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import { OverlayTrigger, Tooltip } from 'react-bootstrap';
+import { connect } from 'react-redux';
+import { Translate } from 'react-redux-i18n';
+
+import * as actions from '../../../actions/adminActions/resourcesCenter';
+import EditResourceForm from './editResourceForm';
+
+export const createResourceTooltip = (
+  <Tooltip id="createResourceTooltip">
```

Comment:
Same here, add it to `components/common/tooltips.jsx`

Reply:
done